kong/tools/wrpc/proto.lua

```diff
@@ -156,19 +156,15 @@ end
 -- If calling the same method with the same args several times,
 -- (to the same or different peers), this method returns the
 -- invariant part, so it can be cached to reduce encoding overhead
-function _M:encode_args(name, ...)
+function _M:encode_args(name, arg)
```

Comment:
this now supports a single argument, better rename to `:encode_arg`

Reply:
I think it's still `args`, but all arguments for 1 call.